src/vaadin-combo-box-mixin.d.ts

```diff
@@ -63,7 +63,7 @@ interface ComboBoxMixin {
    * can be assigned directly to omit the internal filtering functionality.
    * The items can be of either `String` or `Object` type.
```

Comment:
> The items can be of either `String` or `Object` type.
This is now inconsistent with the type definitions, is not it? Should we update the comment or the type definitions?

Reply:
I could not get TypeScript to work properly using `string | object` neither `string | Record<string, unknown>`.

IMO we could keep the comment as it describes the way component is designed to work.